server/task_executor_schema_update_ghost_sync.go

```diff
@@ -203,14 +247,36 @@ func executeGhost(instance *api.Instance, databaseName string, statement string)
 		user:           adminDataSource.Username,
 		password:       adminDataSource.Password,
 		database:       databaseName,
+		table:          tableName,
 		alterStatement: statement,
+		socketFilename: getSocketFilename(task.ID, task.Database.ID, databaseName, tableName),
 		noop:           false,
+		// serverID should be unique
+		serverID: 10000000 + uint(task.ID),
 	})
 	if err != nil {
 		return fmt.Errorf("failed to init migrationContext for gh-ost, error: %w", err)
 	}
 
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
This `ctx` is useful if we `migrator.Migrate()` goes wrong. The go routine where we watch `migrationContext.IsPostponingCutOver` would go on forever otherwise.

Reply:
SG.